
date: 2002/12/03 04:30:01;  author: rob;  state: Exp;  lines: +0 -27
treat all messages equally (or, save headlines to offline storage) [atrus@rifetech.com]

Index: jsm/modules/mod_filter.c
===================================================================
RCS file: /home/cvs/jabberd14/jsm/modules/mod_filter.c,v
retrieving revision 1.26
retrieving revision 1.27
diff -u -r1.26 -r1.27
--- jsm/modules/mod_filter.c	3 Jul 2002 12:05:15 -0000	1.26
+++ jsm/modules/mod_filter.c	3 Dec 2002 04:30:01 -0000	1.27
@@ -125,20 +125,6 @@
 {
     xmlnode cur;
 
-    /* only store normal, error, or chat */
-    if(m->packet->type == JPACKET_MESSAGE)
-    {
-        switch(jpacket_subtype(m->packet))
-        {
-        case JPACKET__NONE:
-        case JPACKET__ERROR:
-        case JPACKET__CHAT:
-            break;
-        default:
-            return;
-        }
-    }
-
    /* look for event messages */
     for(cur = xmlnode_get_firstchild(m->packet->x); cur != NULL; cur = xmlnode_get_nextsibling(cur))
         if(NSCHECK(cur, NS_EVENT))
@@ -335,19 +321,6 @@
     jp = m->packet;
     if(m->packet->type == JPACKET_PRESENCE)
         return M_IGNORE;
-
-    if(m->packet->type == JPACKET_MESSAGE)
-    {
-        switch(jpacket_subtype(m->packet))
-        {
-        case JPACKET__NONE:
-        case JPACKET__ERROR:
-        case JPACKET__CHAT:
-            break;
-        default:
-            return M_PASS;
-        }
-    }
 
     if(m->user == NULL) 
         return M_PASS;
Index: jsm/modules/mod_offline.c
===================================================================
RCS file: /home/cvs/jabberd14/jsm/modules/mod_offline.c,v
retrieving revision 1.18
retrieving revision 1.19
diff -u -r1.18 -r1.19
--- jsm/modules/mod_offline.c	8 Feb 2002 07:44:17 -0000	1.18
+++ jsm/modules/mod_offline.c	3 Dec 2002 04:30:02 -0000	1.19
@@ -58,16 +58,6 @@
         return M_HANDLED;
     }
 
-    switch(jpacket_subtype(m->packet))
-    {
-    case JPACKET__NONE:
-    case JPACKET__ERROR:
-    case JPACKET__CHAT:
-        break;
-    default:
-        return M_PASS;
-    }
-
    /* look for event messages */
     for(cur = xmlnode_get_firstchild(m->packet->x); cur != NULL; cur = xmlnode_get_nextsibling(cur))
         if(NSCHECK(cur,NS_EVENT))
